internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -223,6 +232,27 @@ func resourceKustoCluster() *pluginsdk.Resource {
 					return commonschema.ZonesMultipleOptionalForceNew()
 				}(),
 
+				"allowed_ip_range_list": {
```

Comment:
Rename to https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/search/search_service_resource.go#L116